code/modules/client/preferences/ambient_occlusion.dm

```diff
@@ -10,3 +10,9 @@
 		return
 
 	plane_master.backdrop(client.mob)
+
+	var/atom/movable/screen/plane_master/game_world/fov_hidden/plane_master_fov = locate() in client?.screen
+	if (!plane_master_fov)
+		return
+
+	plane_master_fov.backdrop(client.mob)
```

Comment:
What does this have to do with ambient occlusion?

Reply:
It creates a new plane which requires the ambient occlusion to mimick looking just like the game plane. If there is a way to bundle up the planes and apply that backdrop together to them then I'll do it, but I do not know how or whether it's possible. I'm not familiar with the new rendering capabilities after the refactor.